backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/spi/ElasticsearchBackendImplSettings.java

```diff
@@ -22,4 +27,37 @@ private ElasticsearchBackendImplSettings() {
 
 	public static final String CLIENT_FACTORY = "client_factory";
 
+	/**
+	 * The prefix expected for the key of every Hibernate Search configuration property.
+	 */
+	public static final String PREFIX = EngineSettings.PREFIX + "backend.";
+
+	/**
+	 * The {@link ElasticsearchWorkExecutorProvider} used to create work executors.
+	 * <p>
+	 * Expects a reference to a bean of type {@link ElasticsearchWorkExecutorProvider}.
+	 * <p>
+	 * Defaults to {@link Defaults#BACKEND_WORK_EXECUTOR_PROVIDER}.
+	 */
+	public static final String BACKEND_WORK_EXECUTOR_PROVIDER = PREFIX + Radicals.BACKEND_WORK_EXECUTOR_PROVIDER;
```

Comment:
Shouldn't this be in SPI settings rather than impl settings? It's not just for our own benefit or for tests, it's supposed to be used by integrators. Well, we don't have any at the moment, but still... It might be useful in Quarkus, for example.

Reply:
So if I got it correctly - just add another settings class next to this one but name it properly, right?

Reply:
on it 😄

Reply:
and done